src/main/java/project/cinema/service/UserService.java

```diff
@@ -7,4 +7,6 @@
     User add(User user);
 
     Optional<User> findByEmail(String email);
+
+    Optional<User> getById(Long id);
```

Comment:
```suggestion
    User getById(Long id);
```
let's return the entity from this method

Reply:
fix